tests/test_interventions.py

```diff
@@ -1203,7 +1236,148 @@ def test_quarantined_have_trace_token(self, test_params, time_steps_test ):
+
+        # count the number of people who are quarantined from each traced list
+        df_indiv = pd.read_csv( constant.TEST_INDIVIDUAL_FILE, comment="#", sep=",", skipinitialspace=True )
+        df_quar  = df_indiv.loc[:,["ID","quarantined"]]
+        df_quar  = pd.merge( pre_df_trace, df_indiv, left_on = "traced_ID", right_on = "ID", how = "left" )
+        df_quar  = df_quar[ df_quar["quarantined"] == True ].groupby("index_ID").size().reset_index(name="n_quar")
+        df_trace_sum = pd.merge( pre_df_symp, df_quar, on = "index_ID", how = "left")
+
+        priority_not_traced = df_trace_sum[ ( df_trace_sum["priority"]==True) & ( df_trace_sum["n_quar"] != df_trace_sum["n_interactions"])]
+        np.testing.assert_equal(len(df_trace_sum[(df_trace_sum["priority"]==True)]) > 30, True, "In-sufficient priority cases to test")
+        if ( ( params.get_param( "test_order_wait_priority" ) != -1 ) and ( params.get_param( "test_result_wait_priority" ) != -1 ) ):
+            np.testing.assert_equal(len(priority_not_traced), 0, "Traced people not quarantined immediately following a priority test")
+
+        # get the people who have been uniquely traced by this index and see how many are quarantined
+        df_trace = pd.read_csv( constant.TEST_TRACE_FILE, comment="#", sep=",", skipinitialspace=True )
+        df_trace = df_trace.groupby("traced_ID").size().reset_index(name="n_traced")
+        df_trace = pd.merge( pre_df_trace, df_trace, on = "traced_ID", how = "left" )
+        df_trace = df_trace[ df_trace["n_traced"]==1]
+        df_quar  = df_indiv.loc[:,["ID","quarantined"]]
+        df_quar  = pd.merge( df_trace, df_indiv, left_on = "traced_ID", right_on = "ID", how = "left" )
+        df_quar  = df_quar[ df_quar["quarantined"] == True ].groupby("index_ID").size().reset_index(name="n_quar_uniq")
+        df_trace_sum = pd.merge( df_trace_sum, df_quar, on = "index_ID", how = "left")
+        df_trace_sum.fillna(0, inplace=True)
+          
+        non_priority_traced = df_trace_sum[ ( df_trace_sum["priority"]==False) & ( df_trace_sum["n_quar_uniq"] > 1 )]
+        np.testing.assert_equal( sum( df_trace_sum["priority"]==False ) > 30, True, "Insufficient non-priority cases to test" )
+        np.testing.assert_equal( len( non_priority_traced ), 0, "Non-priority index cases traced people quarantined too soon" )
+        
+        # do a dirty measure rough measure as well to make sure not all are multiply traced
+        non_priority_traced2 = df_trace_sum[ ( df_trace_sum["priority"]==False ) & ( df_trace_sum["n_interactions"] == df_trace_sum["n_quar"] ) ]
+        np.testing.assert_equal( len( non_priority_traced2 ), 0, "Non-priority index cases traced people quarantined too soon (rough measure)" )
+
+        # next step forward to when the non-priority test results are expected
+        for time in range( params.get_param( "test_order_wait" ) + params.get_param( "test_result_wait" ) - params.get_param( "test_result_wait_priority" )):
```

Comment:
I think this should have a  `-params.get_param( "test_order_wait_priorty")` as well. Without I expect it fails the test case `test_order_wait=2` and `test_order_wait_priority = 1`.